src/disc-golf-events/DiscGolfEventDetails.tsx

```diff
@@ -0,0 +1,136 @@
+
+        const url = `${config.discGolfServiceUrl}/public/events/${id}`;
+
+        fetch(url, {
+            method: 'GET',
+            credentials: 'include',
+            headers: {
+                'Content-Type': 'application/json'
+            }
+        })
+            .then(res => {
+                if (!res.ok) {
+                    throw new Error(`No tournament found (status: ${res.status})`);
+                }
+                return res.json();
+            })
+            .then((data: DiscGolfEvent) => {
+                setEvent(data);
+                setLoading(false);
+            })
+            .catch(err => {
+                console.error('Tournament download error:', err);
+                setError(err.message || 'The tournament data could not be downloaded.');
+                setLoading(false);
+            });
+    }, [id]);
+
+    useEffect(() => {
+        fetchEventDetails();
+    }, [fetchEventDetails]);
+
+    if (loading) {
+        return (
+            <Container maxWidth="md" sx={{ mt: 5, display: 'flex', justifyContent: 'center' }}>
+                <CircularProgress />
+            </Container>
+        );
+    }
+
+    if (error) {
+        return (
+            <Container maxWidth="md" sx={{ mt: 5 }}>
+                <Alert severity="error">{error}</Alert>
+            </Container>
+        );
+    }
+
+    if (!event) {
+        return (
+            <Container maxWidth="md" sx={{ mt: 5 }}>
+                <Alert severity="warning">Tournament not found</Alert>
+            </Container>
+        );
+    }
+
+    return (
+        <Container maxWidth="md" sx={{ mt: 4, mb: 4 }}>
+            <Button
+                variant="outlined"
+                onClick={() => navigate('/')}
+                sx={{ mb: 2 }}
+            >
+                ⬅️ Back to Events
+            </Button>
+
+            <Paper elevation={3} sx={{ p: 4 }}>
+                <Typography variant="h4" gutterBottom>
+                    {event.tournamentTitle}
+                </Typography>
+
+                <Typography variant="h6" gutterBottom>
+                    Region: {event.region}
+                </Typography>
+
+                <Typography variant="body1" sx={{ mt: 2 }}>
+                    Tournament date: <strong>{new Date(event.tournamentDate).toLocaleDateString('pl-PL')}</strong>
+                </Typography>
+
+                {event.pdga && (
+                    <Typography variant="body1" sx={{ mt: 2 }}>
+                        PDGA: {event.pdga}
+                    </Typography>
+                )}
+
+                {event.externalLink && (
+                    <div style={{ marginTop: '16px' }}>
+                        {event.externalLink.split(';').filter(link => link.trim() !== '').map((link, index, array) => (
+                            <Typography key={index} variant="body2" sx={{ mt: 1 }}>
+                                <a
+                                    href={link.trim()}
```

Comment:
And here for sure no need to trim(), you just filtered it above. Let's stop using trim() at all - otherwise we will have hundreds of trim's.